libexec/scoop-search.ps1

```diff
@@ -4,119 +4,187 @@
 #
 # If used with [query], shows app names that match the query.
 # Without [query], shows all the available apps.
-param($query)
-. "$psscriptroot\..\lib\core.ps1"
-. "$psscriptroot\..\lib\buckets.ps1"
+param(
+    [Parameter(Mandatory = $true)]
+    $Query,
+    [Switch] $Remote
```

Comment:
Missing documentation for this parameters.

Why is this parameter needed? I would expect that if this is specified ONLY remotes are searched. But locals are searched all the time.

Reply:
Query is now mandatory. Change documentation for help command, reflecting this change.